.github/workflows/ci.yaml

```diff
@@ -76,8 +76,18 @@ jobs:
       run: ./stack build --test --only-dependencies
     - name: Build
       run: ./stack build --fast --test --no-run-tests ${{ matrix.project }}
+      if: matrix.project != 'pgt'
+    - name: Build pgt binary
+      run: ./stack build --fast --test --no-run-tests --copy-bins --flag pgt:build-executable ${{ matrix.project }}
+      if: matrix.project == 'pgt'
     - name: Run tests
       run: ./stack build --fast --test ${{ matrix.project }}
+    - name: Add pgt binary artifact
+      uses: actions/upload-artifact@v3
+      with:
+        name: pgt
```

Comment:
__comment:__ I am going to name this `pgt-linux-amd64` so it is clear *this* produces a linux binary.

At some point we should probably produce macos binaries too.